src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -85,7 +85,7 @@ export class SearchMenu extends React.Component {
     // end early / clear suggestions if user input is too short
     if (userInput?.length <= 2) {
       if (this.state.suggestions.length > 0) {
-        this.setState({ suggestions: [] });
+        this.setState({ suggestions: [], savedSuggestions: [] });
```

Comment:
What does this do?

Reply:
This clears out all of the suggestions is the user's input is below a certain threshold!